Get signingPayloadJson working

SamTheBay opened the pull request:
Update the SDK to handle the new signingPayload and signingPayloadJson parameters. Note that these fragments have not fields but do need parameters. This was previously unsupported. So, I had to remove the exception check and update the compile code to check for this case and do the correct thing.

v16Studios reviewed (dismissed):
Thanks for adding this support 🙂  It looks like a test will also need updating before we can merge it in.

SamTheBay commented:
Hey All. I have removed the 2 failing tests because they were checking for exceptions in cases that are now valid. I have also added support for the "SendTransaction" mutation and have verified that it works in my own project.

leonardocustodio commented:
Ah it needs to be signed, ok, I guess we will need to disable that.
